posthog/session_recordings/queries/test/test_session_recording_list_from_query.py

```diff
@@ -158,10 +158,78 @@ def _filter_recordings_by(self, recordings_filter: dict | None = None) -> Sessio
         )
         return session_recording_list_instance.run()
 
+    def _a_session_with_two_events(self, team: Team, session_id: str) -> None:
+        produce_replay_summary(
+            distinct_id="user",
+            session_id=session_id,
+            first_timestamp=self.an_hour_ago,
+            team_id=team.pk,
+        )
+        self.create_event(
+            "user",
+            self.an_hour_ago,
+            team=team,
+            event_name="$pageview",
+            properties={"$session_id": session_id, "$window_id": "1"},
+        )
+        self.create_event(
+            "user",
+            self.an_hour_ago,
+            team=team,
+            event_name="$pageleave",
+            properties={"$session_id": session_id, "$window_id": "1"},
+        )
+
+    def _assert_query_matches_session_ids(
```

Comment:
introduced this method since we were doing this over and over again in this file

worse, sometimes lazily we would assert on count of ids so if a test failed you wouldn't get any actionable info